jupyterhub/Dockerfile

```diff
@@ -34,3 +34,9 @@ COPY jupyterhub_config.py /srv/jupyterhub/
 COPY wait-for-postgres.sh /srv/jupyterhub/
 
 CMD ["./wait-for-postgres.sh", "jupyterhub", "--config", "/srv/jupyterhub/jupyterhub_config.py"]
+
+USER root
+
+COPY illumidesk-80.png /usr/local/share/jupyterhub/static/images/illumidesk-80.png

```

Comment:
The image currently has the icon in the images directory but does not exist in the `srv/jupyterhub` directory. I believe this should be reversed.
`COPY /usr/local/share/jupyterhub/static/images/illumidesk-80.png illumidesk-80.png `

Reply:
I was under the understanding that the config in the helm chart uses the `/usr/local/share/jupyterhub/static/images/illumidesk-80.png` path.

Reply:
@Abhi94N just FYI, the [COPY directive](https://docs.docker.com/engine/reference/builder/) for Dockerfiles copies from the directory (context) where the Dockerfile is located.